statvar_imports/ndap/nss_health/pvmap/metadata_seven.csv

```diff
@@ -0,0 +1,17 @@
+parameter,value
+url,https://ndap.niti.gov.in/dataset/7314?tab=data
+description,2018 average medical expenditure and non medical expenditure
+#place_type,"Country, State"
+#places_within,country/IND
+start_date,2017
+end_date,2017
+release_frequency,annual
+process,
+comments,
+output_columns,"observationAbout,observationDate,value,variableMeasured,unit,scalingFactor"
+mapped_rows,1
+mapped_columns,7
+header_rows,1
+drop_statvars_without_svobs,0
+places_resolved_csv,https://docs.google.com/spreadsheets/d/1PhulWztRK8xbARAGPEoaIRe7ZbFq-KT_HX9RQ9BZBak/edit?gid=1018588371#gid=1018588371

```

Comment:
pls replace reference to http://docs with a local csv file, such as place_resolved.csv